packages/fetch/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@effection/fetch",
+  "version": "0.0.1",
+  "description": "Fetch operation for Effection",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "repository": "https://github.com/thefrontside/effection.git",
+  "author": "Frontside Engineering <engineering@frontside.io>",
+  "license": "MIT",
+  "files": [
+    "dist/*",
+    "README.md"
+  ],
+  "scripts": {
+    "lint": "eslint '{src,test}/**/*.ts'",
+    "prepack": "tsc --outdir dist --declaration --sourcemap",
```

Comment:
Let's use tsdx as we do in other packages like effection/event & effection/node

Reply:
Done